src/components/Heading.js

```diff
@@ -1,16 +1,24 @@
 import styled from 'styled-components/macro';
 
-export default function Heading({ children }) {
-  return <Header>{children}</Header>;
+export default function Heading({ title }) {
+  return (
+    <>
+      <Header>{title}</Header>
+    </>
+  );
 }
 
 const Header = styled.h1`
   display: grid;
   place-items: center;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  width: 100vw;
```

Comment:
100vw shouldn´t be used like we learned in the public code review. Doesn´t 100% work here?